### ISSUE TITLE
Feat : Build the authentication for this app

### ISSUE THREAD

**Issue by ghost** [ishan301/VerveChat-frontend#4]
I can build these and many features , let's collaborate

**Comment by ishan301** [comment]
Sure. Let's collaborate. You can build the authentication for the app. We will be using Google auth to log in and using email and password to sign in. The data for the users will be stored in the backend. The repo for the same can be found here [VerveChat-Backend](https://github.com/ishan301/verve-chat-backend)

**Comment by ghost** [comment]
Okay

**Comment by ghost** [comment]
This will Require Much More Work

**Comment by ishan301** [comment]
Yes. I will be building it now. You can share your progress if you made anything.
